server/src/modules/actions/documentContent.actions.ts

```diff
@@ -8,7 +8,7 @@ type TCreateDocumentContent = Omit<IDocumentContent, "id">;
 export const createDocumentContent = async (data: TCreateDocumentContent) => {
   const { insertedId: _id } = await getDbCollection(
     "documentContents"
-  ).insertOne(data);
+  ).insertOne({ ...data, updated_at: new Date(), created_at: new Date() });
```

Comment:
Le probleme ici c'est que `updated_at` et `created_at` ne seront pas egaux a quelques microsecondes mais ça peut etre problematique pour les query mongodb.

Le probleme ici c'est que `updated_at` et `created_at` ne seront pas egaux a quelques microsecondes mais ça peut etre problematique pour les query mongodb.

```suggestion
 const now = new Date();
  const { insertedId: _id } = await getDbCollection(
    "documentContents"
  ).insertOne({ ...data, updated_at: now, created_at: now ) });```